CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+All notable changes to this project will be documented in this file, per the [Keep a Changelog standard](http://keepachangelog.com/), and will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased] - TBD
+
+## [0.1.0] - 2023-03-07
+- Initial release.
+
+[Unreleased]: https://github.com/10up/cypress-wp-utils/compare/trunk...develop
```

Comment:
we only have `trunk`, should we branch after 0.1.0 a `develop` branch and set that as the default branch?